tests/ActiveRelationTest.php

```diff
@@ -49,7 +57,7 @@ public function testFindEager()
     public function testFindCompositeLink()
     {
         $articles = ArticleIndex::find()->with('sourceCompositeLink')->all();
-        $this->assertEquals(2, count($articles));
+        $this->assertEquals($this->pagination->defaultPageSize, count($articles));
```

Comment:
This is not a proper assertion.
Exact number match should be used as it was before.